.changelog/3507.txt

```diff
@@ -0,0 +1,4 @@
+```release-note:bug
+docs: Add ability to automatically put categories with no required fields in 
+the optional section of plugin docs. 
+```
```

Comment:
So our changelog is specifically for user-facing features. Since the docs cli is for the waypoint team, I'd remove this changelog!